regression/library/test_manycore_vcache_sequence.c

```diff
@@ -70,22 +70,22 @@ int test_manycore_vcache_sequence() {
                 bsg_pr_test_info("%s: Testing DRAM bank (%" PRIu32 ",%" PRIu32 ")\n",
                                  __func__, dram_coord_x, dram_coord_y);
                 
-                for (size_t i = 0; i < ARRAY_LEN; i++) {
+                for (size_t i = 0; i < ARRAY_LEN; i += 1) {
                         if ((i % 64) == 1)
                                 bsg_pr_test_info("%s: Have written and read back %4zu words\n",
                                                  __func__, i);
                         
                         hb_mc_npa_t npa = hb_mc_npa_from_x_y(dram_coord_x,
                                                              dram_coord_y,
-                                                             BASE_ADDR+i);
+                                                             BASE_ADDR+(i << 2));
```

Comment:
I think this is a pretty minor niggle, but it would be nice if this computation `(i<<2)` were done in only one place.